estuary/utils/recents.py

```diff
@@ -22,7 +22,7 @@ def get_recent_nodes():
     :rtype: tuple
     """
     timestamp_dict = {
-        FreshmakerEvent.__label__: 'id',
+        FreshmakerEvent.__label__: 'time_done',
```

Comment:
I think it's fine to use `time_done`, but this would limit it to just `FreshmakerEvent` nodes that have completed. Is that okay?

Reply:
@mprahl I thought about that too. I wonder if using the time it was created would be better?

Reply:
@sarah256 that's a good idea. We can change it later if needed.